themes/OpenTermsArchive/layouts/index.html

```diff
@@ -0,0 +1,9 @@
+{{ define "main" }}
```

Comment:
> This seems to be a duplicate of `themes/OpenTermsArchive/layouts/_default/single.html`

Reply:
Yes: `index.html` is the template layout for the homepage, `single.html` is the one for the pages that will be added and they actually use the same layout. Previously, in the frontmatter of the homepage, I had specified the layout `single`, but it is better to minimize the complexity of the frontmatter and have two identical files for now.